packages/lodestar/README.md

```diff
@@ -1,6 +1,5 @@
 # Lodestar
 
-[![](https://img.shields.io/travis/com/ChainSafe/lodestar/master.svg?label=master&logo=travis "Master Branch (Travis)")](https://travis-ci.com/ChainSafe/lodestar)
 [![Discord](https://img.shields.io/discord/593655374469660673.svg?label=Discord&logo=discord)](https://discord.gg/aMxzVcr)
 ![ETH2.0_Spec_Version 0.12.1](https://img.shields.io/badge/ETH2.0_Spec_Version-0.12.1-2e86c1.svg)
```

Comment:
Looks like there's still some stale badges

Reply:
do you think we should remove that eth2 spec version badge?  on one hand, we'd have to update it every so often, but on the other hand, it might be a good thing to keep us in check and make sure we actually are up to spec.  if we wanna keep it, i can go through the past version tags in https://github.com/ethereum/eth2.0-specs/tags and make sure we are up to spec.

Reply:
I think we could just bump them all to 1.0.0, and once the altair release is locked down, then bump to 1.1.0